CODE/cgi-bin/mc3.pl

```diff
@@ -1330,12 +1330,24 @@ =head1 HYPOCENTERS FILES
 
 		# --- link to Sefran screenshot
 		$html .= "<td>";
-		#FB-was: my $sefranPNG = "$evt_annee4/$MC3{PATH_IMAGES}/$evt_annee4$evt_mois/$MC3{FILE_PREFIX}$png";
-		my $sefranPNG = "$evt_annee4/$MC3{PATH_IMAGES}/$evt_annee4$evt_mois/$png";
-		if (-f "$MC3{ROOT}/$sefranPNG") { 
-			#$html .= "<a href=\"$MC3{PATH_WEB}/$sefranPNG\" data-lightbox=\"$sefranPNG\" title=\"$sefranPNG\" onMouseOut=\"nd()\" onMouseOver=\"overlib('$imagePOPUP',CAPTION,'$imageCAPTION')\"><img src=\"$amplitude_img\" border=\"0\"></a>";
-			$html .= "<img border=\"0\" wolbsrc=\"$MC3{PATH_WEB}/$sefranPNG\" src=\"$amplitude_img\" onMouseOut=\"nd()\" onMouseOver=\"overlib('$imagePOPUP',CAPTION,'$imageCAPTION')\">";
+		#FB-was: my $event_img_subdir = "$evt_annee4/$MC3{PATH_IMAGES}/$evt_annee4$evt_mois/$MC3{FILE_PREFIX}$event_img";
+		my $event_img_subdir = "$evt_annee4/$MC3{PATH_IMAGES}/$evt_annee4$evt_mois";
+		my $event_img_path = "$MC3{ROOT}/$event_img_subdir/$event_img";
+
+		# Split the MC3 column value on commas in case multiple images were to be displayed
+		my @img_list = map { $_ =~ s/^\s+|\s+$//g; $_; } split(/,/, "$event_img");
+
+		if (@img_list) {
+			# Define the icon visible in the MC3 'Sefran' column
+			# (wolbtarget designates the gallery of images to display defined below)
+			$html .= "<img wolbtarget=\"event-img-$id_evt\" border=\"0\" src=\"$amplitude_img\" onMouseOut=\"nd()\" onMouseOver=\"overlib('$imagePOPUP',CAPTION,'$imageCAPTION')\">";
```

Comment:
You can add to this `<img>` tag the following style attribute `style="cursor:pointer;"` in order to have to cursor changing to a hand, which makes it clear that the image is clickable.

Reply:
done in commit https://github.com/IPGP/webobs/pull/53/commits/b93179fc78c87a396df2ecc3f81737bb8acb5c86